sdk/logs/src/main/java/io/opentelemetry/sdk/logs/ReadWriteLogRecord.java

```diff
@@ -23,7 +24,23 @@ public interface ReadWriteLogRecord {
    */
   <T> ReadWriteLogRecord setAttribute(AttributeKey<T> key, T value);
 
-  // TODO: add additional setters
```

Comment:
Let's keep this TODO. There's still no setters for mutating the other fields on a LogRecord, like timestamp, severity, body, etc.